main.go

```diff
@@ -37,8 +37,7 @@ func main() {
 	flag.Parse()
 
 	opts := providerserver.ServeOpts{
-		// TODO(dzmitry.kishylau): Update this string with the published name of Retool provider.
-		Address: "registry.terraform.io/retool/retool",
+		Address: "registry.terraform.io/tryretool/retool",
```

Comment:
Tbh, not sure what this string does exactly, but changing it to `tryretool/retool` breaks local dev for unknown reasons.